mbbsd/register.c

```diff
@@ -597,6 +597,17 @@ do_register_captcha()
 #endif
 }
 
+static void
+check_register_connection_security()
+{
+#ifdef REQUIRE_SECURE_CONN_TO_REGISTER
+    if (mbbsd_is_secure_connection())
+	return;
+    vmsg("�ШϥΦw���s�u��U�b��!");
+    exit(1);
+#endif
+}
+
```

Comment:
The function is only used once, and does not help with readability. Just inline it.

Reply:
OK. Inlined.